external/qjs_webcrypto_module.c

```diff
@@ -3955,7 +3954,7 @@ qjs_webcrypto_sign(JSContext *cx, JSValueConst this_val, int argc,
         ret = JS_NewBool(cx, rc != 0);
     }
 
-    return qjs_webcrypto_result(cx, ret, 0);
```

Comment:
If `ret` is `JS_EXCEPTION`, then here is:
```
return qjs_webcrypto_result(cx, JS_EXCEPTION, 0);
```
I'm confused about the value of the (3) parameter, 0. It looks like an issue?

Reply:
We check for exceptions in the code manually and go to `fail` code blocks.
So, the third argument should consistent with a valid (non-exception) value.